index.js

```diff
@@ -79,7 +79,7 @@ var Sqlite3Driver = Base.extend({
       constraint.push('UNIQUE');
     }
 
-    if (spec.defaultValue) {
+    if (spec.defaultValue !== undefined) {
```

Comment:
Given the docs, there is no empty DEFAULT constraint:
![](https://www.sqlite.org/images/syntax/column-constraint.gif)